fluent-plugin-enhance-k8s-metadata/test/helper.rb

```diff
@@ -29,8 +29,12 @@ def stub_apis
     )
   stub_request(:get, %r{/api/v1$})
     .to_return(body: test_resource('api_list_core_v1.json'), status: 200)
+  stub_request(:get, %r{/apis/apps$})
+    .to_return(body: test_resource('api_list_apps.json'), status: 200)
   stub_request(:get, %r{/apis/apps/v1$})
     .to_return(body: test_resource('api_list_apps_v1.json'), status: 200)
+  stub_request(:get, %r{/apis/extensions$})
+    .to_return(body: test_resource('api_list_extensions.json'), status: 200)
```

Comment:
shouldn't we replace something here instead of adding new endpoint?

Reply:
done